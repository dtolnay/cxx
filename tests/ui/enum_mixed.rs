#[cxx::bridge]
mod ffi {
    enum Bad {
        A(i32),
        B = 1,
    }
}

fn main() {}
