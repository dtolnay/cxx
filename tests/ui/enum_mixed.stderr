error: Mixed c-style enums with Rust enums
 --> tests/ui/enum_mixed.rs:3:5
  |
3 | /     enum Bad {
4 | |         A(i32),
5 | |         B = 1,
6 | |     }
  | |_____^
